custom_components/alexa_media/__init__.py

```diff
@@ -426,12 +426,19 @@ def hide_serial(item):
                       exclude_filter)
 
         if new_alexa_clients:
+            cleaned_config = config.copy()
+            cleaned_config.pop(CONF_SCAN_INTERVAL, None)
+            # CONF_SCAN_INTERVAL causes a json error in the recorder because it
+            # is a timedelta object.
+            cleaned_config.pop(CONF_PASSWORD, None)
```

Comment:
Thank you for taking care of this.   Would it make sense to pop this out of `config` (or zero it out) as soon as its known that it's not needed?

Reply:
It'd only partially help I think. The config is passed around HA and I haven't investigated whether components are truly sandboxed from the config.